src/Main.hx

```diff
@@ -322,10 +320,10 @@ class Main {
 				var packageObj = haxe.Json.parse(packageJson);
 				var dependencies: DynamicAccess<String> = packageObj.dependencies != null ? packageObj.dependencies : {};
 				var devDependencies: DynamicAccess<String> = packageObj.devDependencies != null ? packageObj.devDependencies : {};
-				var allDependencies = dependencies.keys().concat(devDependencies.keys());
```

Comment:
allDependencies might be better here to remind me what this was about (moduleNames gives less info)